nvflare/fuel/hci/client/api.py

```diff
@@ -488,8 +488,10 @@ def do_command(self, command):
         elif len(entries) == 1:
             self.set_command_result(None)
             ent = entries[0]
-            ent.handler(args, self)
+            return_result = ent.handler(args, self)
```

Comment:
when will this be used?

what client command has a return result?

Reply:
this is for the client side commands, so the File transfer, but now HA, the result is returned.

Reply:
Got it, the "Admin client" side commands.